index.html

```diff
@@ -442,8 +442,14 @@
         <li>If <var>timeRemaining</var> is negative, set it to 0.</li>
         <li>Return <var>timeRemaining</var>.</li>
       </ol>
-      <p>The <code><dfn>didTimeout</dfn></code> attribute on an `IdleDeadline`
-      object MUST return `true` if the callback was invoked by the <a>invoke idle callback timeout algorithm</a>, and `false` otherwise.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <dfn>timeout</dfn>, which is
+      initially false. The <code><dfn>didTimeout</dfn></code> getter MUST
+      return <a>timeout</a>.</p>
+      <p class='note'>The <a>invoke idle callback timeout algorithm</a> sets
+      <a>timeout</a> to true to specify that the callback is being executed
+      outside an idle period, due to it having exceeded the timeout property
```

Comment:
With this update we now have two "timeout" concepts being using interchangeably. Let's make them more explicit in text to minimize confusion?

..."due to it having exceeded the value of the `<a>IdleDeadline</a>'s <a>timeout</a>` attribute"?